### PR TITLE
W/A for nooobaa 4.10 operator installed on ODF 4.9

### PR DESCRIPTION
**Pull request by petr-balogh**
W/A issue:
https://github.com/red-hat-storage/ocs-ci/issues/5812

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

**Comment by petr-balogh** [comment]
Running verification job here:
https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-trigger-vsphere-upi-1az-rhcos-vsan-3m-3w-upgrade-ocs-auto/185/

**Comment by petr-balogh** [comment]
New verification job: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-trigger-vsphere-upi-1az-rhcos-vsan-3m-3w-upgrade-ocs-auto/189/

File: ocs_ci/ocs/resources/storage_cluster.py [hunk 1]
```diff
@@ -582,7 +582,18 @@ def verify_storage_system():
...
+    live_deployment = config.DEPLOYMENT.get("live_deployment")
+    ocp_version = version.get_semantic_ocp_version_from_config()
     ocs_version = version.get_semantic_ocs_version_from_config()
+    if (
+        live_deployment
+        and ocs_version == version.VERSION_4_9
+        and ocp_version == version.VERSION_4_10
+    ):
+        log.warning(
+            "Because of the BZ 2079878, we are skipping storage system validation!"
```
Comment (vavuthu) [inline ocs_ci/ocs/resources/storage_cluster.py:594]: BZ 2079878 is for ODF 4.10 upgrade ( from 4.10 live to internal build ). above validation is for ODF 4.9

Comment (petr-balogh) [reply ocs_ci/ocs/resources/storage_cluster.py:594]: Yes, my mistake when copy paste from wrong tab :). I will update to: 2075422. Thanks

**Review by dahorak — APPROVED** [review]

**Comment by vavuthu** [comment]
> New verification: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-trigger-vsphere-upi-1az-rhcos-vsan-3m-3w-upgrade-ocs-auto/193/

upgrade job is passed. marking as Verified.

**Pull request merged**